governance/tests/test-navigation.sh

```
@@ -0,0 +1,78 @@
+#   §E NAVIGATION.md 内的本仓 markdown 链接全部可解析（断链=红）
+#   §F Makefile 暴露 drift-check 入口（C1 预检命令可发现性）
+# 用法：bash governance/tests/test-navigation.sh（零网络）
+set -uo pipefail
+HERE="$(cd "$(dirname "$0")" && pwd)"
+ROOT="$(cd "$HERE/../.." && pwd)"
+
+PASS=0; FAIL=0
+pass() { PASS=$((PASS+1)); echo "PASS  $1"; }
+fail() { FAIL=$((FAIL+1)); echo "FAIL  $1"; }
+
+cd "$ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)